core/mo-services-impl/ccsds-com-impl/src/test/java/esa/mo/com/impl/provider/ArchiveManagerTest.java

```diff
@@ -0,0 +1,199 @@
+//------------------------------------------------------------------------------
```

Comment:
Ensure proper license header is followed

Reply:
Done